pkg/sql/create_index.go

```diff
@@ -585,6 +590,15 @@ func maybeCreateAndAddShardCol(
 	if err != nil && !columnIsUndefined {
 		return nil, err
 	}
+
+	if !cv.IsActive(clusterversion.Start22_1) {
```

Comment:
Your builds are all failing because for some reason you're not importing `clusterversion`. I'm surprised that your IDE did not add the import for you, actually.

Reply:
oh...I created this pr from the other pr with the backporting tool... crdb is evolving